app/assets/stylesheets/_honeycrisp-compact.scss

```diff
@@ -253,5 +281,9 @@
   .client-navigation {
     @include no-bleed();
   }
+
+  .client-container {
+    width: 760px;
+  }
```

Comment:
The reason to use rem is so that if you change what the value of 1rem is, it changes the value of that rem everywhere. Here, I don't know if we want to do that. -- if we changed the value of a rem, do we want the width of this container to change? Usually for containers, I would say no which is why I usually use pixel widths for containers. However, I guess here we're fine to increase the width of this container should a user have a custom style sheet that increases the value of a rem, so maybe rem is good here.

The grid system as such seems, to me, to be created for layout purposes and being able to put different components next to each other. Here, I really just want a limited width wrapping container at a width dictated by Anu for better scannability of messages and notes.